src/cudamatrix/cu-vector.cc

```diff
@@ -711,6 +711,47 @@ void CuVectorBase<Real>::MulElements(const CuVectorBase<Real> &v) {
 
 template<>
 template<>
+void CuVectorBase<double>::CopyFromVec(const CuVectorBase<double> &src) {
+  KALDI_ASSERT(src.Dim() == dim_);
+#if HAVE_CUDA == 1
+  if (CuDevice::Instantiate().Enabled()) {
+    if (dim_ == 0) return;
+    Timer tim;
+    int dimBlock(CU2DBLOCK);
```

Comment:
Convert CU2BLOCK to CU1DBLOCK (same for line below).

The code is still correct. It's just slower, if you use CU2DBLOCK, here.

Actually, I noticed this same problem exists in the original code base. Still, fixing it should speed the code up.